Gruntfile.js

```diff
@@ -28,7 +28,7 @@ module.exports = function (grunt) {
             'src/system.js',
             'src/wrapper-end.js'
           ],
-          'dist/<%= pkg.name %>-dev.src.js': [
+          'dist/<%= pkg.name %>-dev-strict.src.js': [
```

Comment:
I'm just wondering, since we already separately do the `dev` and `production` builds, where the `dev` build is the one that provides the declarative linking layer (which is then left out of the non-dev build), perhaps we just need to keep it as these two builds and rename `dev` to `strict`. Or do you think it's important to have the non-dev strict build for compatibility?

`dev` is also no longer the right name for the other build as it is no longer used for dev workflows. The best name I can come up with right now is `declarative`?